core/domain/improvements_services_test.py

```diff
@@ -178,39 +180,42 @@ def test_fetch_when_number_of_open_tasks_exceed_single_fetch_limit(self):
 
         self.assertEqual(resolved_task_types_by_state_name, {})
         self.assertItemsEqual(
-            [t.to_dict() for t in tasks], [t.to_dict() for t in open_tasks])
+            [improvements.get_task_dict_with_username_and_profile_picture(
```

Comment:
ditto as in _improvements_domain_test.py_ this should not be tested here but in _improvements_test.py_

Reply:
ditto elsewhere

Reply:
Done.